### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by ruchirchauhan**
Code cleanup

### CODE REVIEW

**Review by gregmedd — CHANGES_REQUESTED** [review]

File: test/coverage/communication/PublisherTest.cpp [hunk 1]
```diff
@@ -10,23 +10,16 @@
...
-#include <up-cpp/communication/Publisher.h>
-#include <up-cpp/datamodel/builder/Payload.h>
-#include <up-cpp/datamodel/serializer/UUri.h>
 #include <uprotocol/v1/uri.pb.h>
 
 #include "UTransportMock.h"
-#include "spdlog/spdlog.h"
 #include "up-cpp/communication/Publisher.h"
-#include "up-cpp/datamodel/builder/UMessage.h"
+#include "up-cpp/communication/Subscriber.h"
```
[the remark this answers is not in the document]
Comment (ruchirchauhan) [reply test/coverage/communication/PublisherTest.cpp:17]: done

File: test/coverage/communication/PublisherTest.cpp [hunk 2]
```diff
@@ -10,23 +10,16 @@
...
-#include "up-cpp/transport/UTransport.h"
 
 using namespace uprotocol::communication;
-using namespace uprotocol::datamodel::builder;
 using namespace uprotocol::v1;
-using ::testing::_;
-using ::testing::Return;
+
 namespace {
 using namespace uprotocol::datamodel::builder;
```
[the remark this answers is not in the document]
Comment (ruchirchauhan) [reply test/coverage/communication/PublisherTest.cpp:24]: Its getting those from other headers included, but this is required for building payload

File: test/coverage/communication/PublisherTest.cpp [hunk 3]
```diff
@@ -70,8 +63,17 @@ class TestPublisher : public testing::Test {
...
+	uprotocol::v1::UMessage capture_msg_;
+
+public:
+	void handleCallbackMessage(const uprotocol::v1::UMessage& message);
 };
 
+void TestPublisher::handleCallbackMessage(
+    const uprotocol::v1::UMessage& message) {
+	capture_msg_ = message;
+}
```
[the remark this answers is not in the document]
Comment (ruchirchauhan) [reply test/coverage/communication/PublisherTest.cpp:75]: got added by mistake

**Review by gregmedd — APPROVED** [review]
LGTM

**Comment by gregmedd** [comment]
One last request: Please revise the commit message to better describe the change. I recommend [these guidelines](https://cbea.ms/git-commit/) when writing commit messages.

**Comment by ruchirchauhan** [comment]
This should be ready to be merged

**Pull request merged**